README.md

```diff
@@ -8,6 +8,11 @@ This project is to fullfill the first milestone of the collaborate project for t
 -   **TEAMWORK.md**: The group teamwork contract, outlining our agreed terms for the project
 -   **troubleshooting-1.Rmd**: A completed class-assigned exercise, which focused on correcting errors in markdown
 -   **troubleshooting-1.md**: The output file of the troubleshooting Rdocument
+-   **.gitignore** is used by git to avoid tracking superfluous files.
+
+## Running the code
+-   **troubleshooting-1.rmd** can be run by opening the file in Rstudio, then clicking "knit". This will re-generate **troubleshooting-1.md** as it appears in this directory.
```

Comment:
I think tidyverse needs to be installed if the user wants to run the code locally, with command `install.packages(“tidyverse”)` in R session @thomasbd14

Reply:
Thank you for the reminder! I have added this in https://github.com/stat545ubc-2023/collaborative-group24/pull/19/commits/949ba3703d955086c3c46a891be726a3e1e79909